src/lib/checkbox/_checkbox-theme.scss

```diff
@@ -73,7 +74,7 @@
     }
 
     .mat-checkbox-label {
-      color: $disabled-color;
+      color: mat-color($foreground, disabled-text);
```

Comment:
It turns out that the contrast of `disabled-text` in dark themes doesn't meet a11y standards.

We should use `secondary-text` here for now.

Reply:
Done.